tslint.yaml

```diff
@@ -6,6 +6,28 @@ linterOptions:
   exclude:
     - node_modules/**
     - '**/*.json'
+    - Tasks/AWSCLI/**
```

Comment:
Please add a TODO comment around this block stating that these tasks will be refactored/lint-cleaned over time and reference the github issue if there is one

Reply:
Added in 90f51e7